http/grpc/src/test/java/io/quarkus/ts/http/grpc/XeparateServerIT.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.ts.http.grpc;
+
+import org.junit.jupiter.api.DisplayName;
+
+import io.grpc.Channel;
+import io.quarkus.test.bootstrap.GrpcService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+@DisplayName("SeparateServer")
+//This test should be the last, or we get complains, that the channel was not shut down before closure. This is a bug in our framework.
+public class XeparateServerIT implements GRPCIT, StreamingHttpIT, ReflectionHttpIT {
```

Comment:
```suggestion
public class SeparateServerIT implements GRPCIT, StreamingHttpIT, ReflectionHttpIT {
```

Reply:
See comment above

Reply:
Thanks, I missed that.

Ad your comment and this workaround: you invoke `app.grpcChannel` therefore it shouldn't be dependable on any other test. I think grpcChannel should be created for this test only. I can't see relation to any other test class.

Reply:
My point is - I only know what you wrote but it seems like a wrong design and we need to fix it, please open issue in Test Framework and link it here, thanks.

Reply:
The issue: https://github.com/quarkus-qe/quarkus-test-framework/issues/1054

The problem is, gRPC java somehow detects, whether the **previous** channel was closed  properly (or not) and complains about that